### PR TITLE
Added comments to RequestedToCapacityRatio formula

### PR DESCRIPTION
**Pull request by jqmichael**
Added comments to RequestedToCapacityRatioResourceAllocation formula to make it easier to follow.

### CODE REVIEW

**Comment by sftim** [comment]
/sig scheduling

@jqmichael it's a good idea to enter a pull request description: this helps reviewers check that the changes you're suggested match your intent

**Comment by jqmichael** [comment]
> /sig scheduling
> 
> @jqmichael it's a good idea to enter a pull request description: this helps reviewers check that the changes you're suggested match your intent

Thanks. Updated the description.

**Comment by jqmichael** [comment]
@sudeshsh

**Comment by ahg-g** [comment]
/lgtm

for sig-scheduling

**Comment by jqmichael** [comment]
@sftim  Do you have bandwidth for a quick look?

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-21067--kubernetes-io-master-staging.netlify.app/docs/concepts/configuration/resource-bin-packing/

File: content/en/docs/concepts/configuration/resource-bin-packing.md [hunk 1]
```diff
@@ -134,21 +134,21 @@ Node Score:
 intel.com/foo  = resourceScoringFunction((2+1),4)
                =  (100 - ((4-3)*100/4)
                =  (100 - 25)
-               =  75
-               =  rawScoringFunction(75)
-               = 7
+               =  75                      # requested + used = 75% * available
+               =  rawScoringFunction(75)
```
Comment (kbhawkey) [inline content/en/docs/concepts/configuration/resource-bin-packing.md:138]: @jqmichael ,
The changes seem OK. What is the file format for the content?

File: content/en/docs/concepts/configuration/resource-bin-packing.md [hunk 2]
```diff
@@ -134,21 +134,21 @@ Node Score:
 intel.com/foo  = resourceScoringFunction((2+1),4)
                =  (100 - ((4-3)*100/4)
                =  (100 - 25)
-               =  75
-               =  rawScoringFunction(75)
-               = 7
+               =  75                      # requested + used = 75% * available
```
Comment (sudeshsh) [inline content/en/docs/concepts/configuration/resource-bin-packing.md:137]: The spacing I was talking about, some have space after "=" while some don't.

Comment (jqmichael) [reply content/en/docs/concepts/configuration/resource-bin-packing.md]: Thanks @sudeshsh. I've fixed the spacing. Please take a quick look when you get a chance.

**Comment by sudeshsh** [comment]
@jqmichael LGTM

**Pull request merged**